server/server.go

```diff
@@ -12,17 +12,22 @@ type rodeServer struct {
 	grafeasClient grafeas.GrafeasV1Beta1Client
 }
 
-func (r *rodeServer) BatchCreateOccurrences(ctx context.Context, occurrenceRequest *grafeas.BatchCreateOccurrencesRequest) (*grafeas.BatchCreateOccurrencesResponse, error) {
+func (r *rodeServer) BatchCreateOccurrences(ctx context.Context, occurrenceRequest *pb.BatchCreateOccurrencesRequest) (*pb.BatchCreateOccurrencesResponse, error) {
 	fmt.Println("This works!!!!!")
 	fmt.Printf("%#v\n", *occurrenceRequest.Occurrences[0])
```

Comment:
I know this isn't the scope of the PR, but can we change this line to something like this 

```
for i := range occurrenceRequest.Occurrences {
		fmt.Printf("%#v\n", *occurrenceRequest.Occurrences[i])
	}
```

Reply:
I'd prefer to remove the print statement entirely, since we only included that for testing purposes.